_includes/parse-server/file-adapters.md

```diff
@@ -175,7 +175,67 @@ var api = new ParseServer({
 
 Don't forget to change **S3_ACCESS_KEY**, **S3_SECRET_KEY** and **S3_BUCKET** to their correct value.
 
-##### S3Adapter constructor options
+### Digital Ocean Spaces
+
+Spaces is an S3-compatible provided by Digital Ocean. It's use the same api as S3 so you can use it with the S3 Adapter.
```

Comment:
Let's keep the descriptions uniform and containing the same info (link to product and docs). Please change to:

> [Digital Ocean Spaces](https://try.digitalocean.com/cloud-storage) is a S3-compatible storage service in the cloud. See their [documentation](https://docs.digitalocean.com/products/spaces/) for more details.